shared/common-adapters/name-with-icon.js

```diff
@@ -46,6 +48,8 @@ const NameWithIconVertical = (props: Props) => {
     >
       {isAvatar && (
         <Avatar
+          editable={props.editableIcon}
+          onEditAvatarClick={props.editableIcon ? props.onEditIcon : undefined}
```

Comment:
its better to use undefined, then the component can do something else. if you pass a function it can't know if its doing something so it won't fallback. aka sometimes it'll wrap itself differently or have different behavior on RN

Reply:
Did not know this, thank you!